source/vibe/inet/webform.d

```diff
@@ -640,3 +641,181 @@ unittest
 	}
 
 }
+
+/**
+ * A MIME entity, typically sent in an HTTP request or e-mail with a "Content-Type"
+ * header value in the form "multipart/*", e.g. "multipart/form-data". Following RFC 2046, this
+ * entity represents one or more different data parts combined into a single body.
+ *
+ * A boundary value preceeded by "--" is used to separate the multipart body parts, and the last
+ * part is indicated by the boundary value also followed by "--". An example HTTP POST request
+ * is shown below:
+ * ```
+ * POST /upload HTTP/1.1
+ * Content-Length: 428
+ * Content-Type: multipart/form-data; boundary=abcde12345
+ * --abcde12345
+ * Content-Disposition: form-data; name="id"
+ * Content-Type: text/plain
+ * 123e4567-e89b-12d3-a456-426655440000
+ * --abcde12345
+ * Content-Disposition: form-data; name="address"
+ * Content-Type: application/json
+ * {
+ *	 "street": "3, Garden St",
+ *	 "city": "Hillsbery, UT"
+ * }
+ * --abcde12345
+ * Content-Disposition: form-data; name="profileImage "; filename="image1.png"
+ * Content-Type: application/octet-stream
+ * {…file content…}
+ * --abcde12345--
+ * ```
+ *
+ * See_Also: https://datatracker.ietf.org/doc/html/rfc2046#section-5.1
+ * See_Also: https://www.w3.org/Protocols/rfc1341/7_2_Multipart.html
+ * See_Also: https://datatracker.ietf.org/doc/html/rfc2388
+ */
```

Comment:
Small code style issue - the multi-line comment style used in the code base is omitting the " * " prefix for the intermediate lines and instead uses a single tab indentation. Also, the first paragraph should just be a single short summary sentence, because it will be listed in the "description" column of the overview tables of the API documentation.